exampleSite/content/posts/creating-a-new-theme.md

```diff
@@ -1,1147 +1,27 @@
 ---
-author: "Michael Henderson"
```

Comment:
I don't think you meant to edit this file.

Reply:
Well spotted, fixed!